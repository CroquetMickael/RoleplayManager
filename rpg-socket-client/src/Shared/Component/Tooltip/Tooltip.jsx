import React from "react";

const Tooltip = ({ text }) => (
  <span className="p-2 mt-10 ml-12 bg-blue-200 rounded tooltip-text">
    {text}
  </span>
);

export { Tooltip };
