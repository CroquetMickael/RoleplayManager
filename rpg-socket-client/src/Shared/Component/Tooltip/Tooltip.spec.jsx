import React from "react";
import { render } from "@testing-library/react";
import {Tooltip} from "./Tooltip";

it("renders correctly", () => {
  const tree = render(<Tooltip>A tooltip</Tooltip>).container;
  expect(tree).toMatchSnapshot();
});
