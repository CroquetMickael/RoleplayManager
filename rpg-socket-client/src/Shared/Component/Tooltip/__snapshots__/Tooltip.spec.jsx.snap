// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <span
    class="p-2 mt-10 ml-12 bg-blue-200 rounded tooltip-text"
  />
</div>
`;
